bin/tachyon

```diff
@@ -16,6 +16,7 @@ function printUsage {
   echo -e "  loadufs       \t Load existing files in underlayer filesystem into Tachyon."
   echo -e "  runTest       \t Run a end-to-end test on a Tachyon cluster."
   echo -e "  runTests      \t Run all end-to-end tests on a Tachyon cluster."
+  echo -e "  jCrashTest    \t Test the Master Journal System in a crash scenario"
```

Comment:
I think it should be separated with runTest because this test needs to start/stop Master.
I'll change it to 'journalCrashTest' and update the 'help'.